### DISCUSSION TITLE
self_pooling vs pooling with a pool vs OG plots

### DISCUSSION THREAD

**Discussion by jayhohoho2019** [Chia-Network/chia-blockchain#8072] [Support]
Hi, I assume technically self pooling and pooling are the same in terms of using portable plots, but self_pooling saves one the effort of providing partial proofs to the pool, but then doesn't get the more predictable rewards distributed from that pool.  Is that the correct understanding? 

Is self pooling then similar to farming with OG plots, except it uses portable plots vs OG plots?

**Comment by keliew** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-FAQ#what-is-a-plot-nft
https://github.com/Chia-Network/chia-blockchain/wiki/Pooling-User-Guide#step-3-create-plot-nft

It's not exactly clear, but it basically means `self_pooling` behaves like og plots, wherein you are not part of a pool and are self-farming. The full rewards are directly to your wallet, although the 1.75 xch need to be _manually_ claimed.

**Comment by BrandtH22** [comment]
@jayhohoho2019 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - yes, you are correct in your understanding. The only other difference is with self-pooling you have to claim the 7/8ths portion from the pool contract whereas OG farming needs no claiming as the rewards are sent directly to the farmer.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
